### DISCUSSION TITLE
config staticman.yml entries

### DISCUSSION THREAD

**Discussion by binarymist** [eduardoboucas/staticman#419] [Q&A]
Can someone please explain what the following configuration values are for:

* githubAuth.clientId
* githubAuth.clientSecret
* githubAuth.required which says to use the undocumented auth.required instead

I've also noticed in [step 4](https://staticman.net/docs/getting-started.html#step-4-hook-up-your-forms) that there is supposed to be a `api-docs` route, but there doesn't appear to be, I've also looked through the [source](https://github.com/eduardoboucas/staticman/blob/master/server.js) and can't find it. Is it hiding somewhere else?

Thanks.

**Comment by VincentTam** [comment]
https://staticman.net/docs/configuration#githubAuth.clientId
dupe of eduardoboucas/staticman.net#14
